packages/plugins/rubygem-the_foreman_proxmox/rubygem-the_foreman_proxmox.spec

```diff
@@ -0,0 +1,116 @@
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name the_foreman_proxmox
```

Comment:
We have a convention naming all plugins `foreman_X`. Would this still be a good time to rename the gem to `foreman_proxmox`? It would make things like installer support easier.

Reply:
Sorry but the gem [foreman_proxmox](https://rubygems.org/gems/foreman_proxmox) is already used, that's why I have to rename it the_foreman_proxmox.